portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -66,13 +70,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
 
     // Gets all existing comments from database
-    List<String> commentsList = new ArrayList<String>();
+    List<Comment> commentsList = new ArrayList<Comment>();
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
     Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
     PreparedQuery results = datastore.prepare(query);
     for (Entity entity : results.asIterable()) {
```

Comment:
If you wanted to get fancy with streams, you could do this:
```java
List<Comment> comments = results.asList().stream().map(Comment::new).collect(toList());
```

Reply:
Will do in separate PR.